### DISCUSSION TITLE
v2.3.0-beta2

### DISCUSSION THREAD

**Discussion by WouterTinus** [simple-acme/simple-acme#6] [Announcements]
(Note: only changes sinces beta1 are listed, check the previous release for more details)

# Features
- A new store plugin is include to generate `.p7b` files for those who need them. Based on [feedback](https://github.com/win-acme/win-acme/discussions/2690) by @mtolentino47 

# Enhancements
- It's now possible to configure the location that the program should use download the [public suffix list](https://publicsuffix.org/) from with the setting `Acme.PublicSuffixListUri`. This also makes it possible to entirely disable updates, by configuring an empty string. Based on [feedback](https://github.com/win-acme/win-acme/discussions/2654) by @Jakl-H.
- @zvillian updated the example script `ImportNTDS.ps1` (https://github.com/simple-acme/simple-acme/pull/3)
- @fardarter added support for user-assigned managed identities in the common Azure library, which means that is now support for both Azure DNS validation and Azure KeyVault storage.
- It's now possible to set the maximum number of host names that is allowed to be included in a single certificate using the setting `Acme.MaxDomains`. This was previously hard-coded to 100 because that is the long-standing limit for Let's Encrypt, but it may be higher for other services or private solutions. Based on [feedback](https://github.com/win-acme/win-acme/discussions/2689) by @rvarma639.

# Code changes
- Configured some automatic checks for pull requests like CodeQL and unit tests

<hr /><em>This discussion was created from the release <a href='https://github.com/simple-acme/simple-acme/releases/tag/v2.3.0.1826'>v2.3.0-beta2</a>.</em>

**Comment by webprofusion-chrisc** [comment]
Hi Wouter, the linux full builds point to the trimmed builds, is that on purpose? Maybe also for linux a snap build would be useful (that's what certbot primarily packages as).
